tests/tck/features/fetch/FetchVertices.strVid.feature

```diff
@@ -444,3 +460,56 @@ Feature: Fetch String Vertices
       GO FROM 'NON EXIST VERTEX ID' OVER serve | FETCH PROP ON team $-
       """
     Then a SyntaxError should be raised at runtime:
+
+  Scenario: format yield
+    When executing query:
+      """
+      FETCH PROP ON * 'Boris Diaw' YIELD id(vertex)
```

Comment:
Why does the `fetch` use vertex/edge while the `subgraph` use vetices/edges? Can we unify these keywords into one so that could reduce the learning cost and confusion for users?

Reply:
The result returned by subgraph is a list type while the result returned by fetch is a vertex type 。So the column names are different。
This can be discussed later, in fact, I think that both are okay

Reply:
I think it's ok now.

Reply:
Change to `edge` and `vertex` for subgraph is more confused.